app/controllers/requests_controller.rb

```diff
@@ -3,19 +3,25 @@
 # crud for tech requests
 class RequestsController < ApplicationController
   def index
+    authorize Request
     @requests = Request.all
   end
 
   def new
+    authorize Request
     @request = Request.new
```

Comment:
Do these models have policies with associated scopes? Even if the scopes just return all records for all users, it might be worth putting the filter in place in case we want to do role-specific filtering.